biodatasets/umnsrs/umnsrs.py

```diff
@@ -48,55 +49,74 @@
-a bar on a touch sensitive computer screen to indicate the degree of similarity or relatedness. The Intraclass
-Correlation Coefficient (ICC) for the reference standard tagged for similarity was 0.47, and 0.50 for relatedness.
-Therefore, as suggested by Pakhomov and colleagues, the subset below consists of 401 pairs for the similarity set and
-430 pairs for the relatedness set which each have an ICC equal to 0.73.
+a bar on a touch sensitive computer screen to indicate the degree of similarity or relatedness.
+The following subsets are available:
+- similarity: A set of 566 UMLS concept pairs manually rated for semantic similarity (e.g. whale-dolphin) using a
+  continuous response scale.
+- relatedness: A set of 588 UMLS concept pairs manually rated for semantic relatedness (e.g. needle-thread) using a
+  continuous response scale.
+- similarity_mod: Modification of the UMNSRS-Similarity dataset to exclude control samples and those pairs that did not
+  match text in clinical, biomedical and general English corpora. Exact modifications are detailed in the paper (Corpus
+  Domain Effects on Distributional Semantic Modeling of Medical Terms. Serguei V.S. Pakhomov, Greg Finley, Reed McEwan,
+  Yan Wang, and Genevieve B. Melton. Bioinformatics. 2016; 32(23):3635-3644). The resulting dataset contains 449 pairs.
+- relatedness_mod: Modification of the UMNSRS-Relatedness dataset to exclude control samples and those pairs that did
+  not match text in clinical, biomedical and general English corpora. Exact modifications are detailed in the paper
+  (Corpus Domain Effects on Distributional Semantic Modeling of Medical Terms. Serguei V.S. Pakhomov, Greg Finley,
+  Reed McEwan, Yan Wang, and Genevieve B. Melton. Bioinformatics. 2016; 32(23):3635-3644).
+  The resulting dataset contains 458 pairs.
 """
 
-_HOMEPAGE = "https://nlp.cs.vcu.edu/data.html#umnsrs"
+_HOMEPAGE = "https://conservancy.umn.edu/handle/11299/196265/"
 
-_LICENSE = "Unknown"
+_LICENSE = "CC0 1.0 Universal"
+
+_BASE_URL = "https://conservancy.umn.edu/bitstream/handle/11299/196265/"
 
 _URLS = {
-    "source": ["https://nlp.cs.vcu.edu/data/similarity-data/UMNSRS_similarity.csv"],
-    "bigbio_pairs": ["https://nlp.cs.vcu.edu/data/similarity-data/UMNSRS_similarity.csv"],
+    "umnsrs_similarity": _BASE_URL + "UMNSRS_similarity.csv",
+    "umnsrs_relatedness": _BASE_URL + "UMNSRS_relatedness.csv",
+    "umnsrs_similarity_mod": _BASE_URL + "UMNSRS_similarity_mod449_word2vec.csv",
+    "umnsrs_relatedness_mod": _BASE_URL + "UMNSRS_relatedness_mod458_word2vec.csv",
 }
 
-_SUPPORTED_TASKS = [
-    Tasks.SEMANTIC_SIMILARITY
-]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+_SUPPORTED_TASKS = [Tasks.SEMANTIC_SIMILARITY]
 
 _SOURCE_VERSION = "1.0.0"
 
 _BIGBIO_VERSION = "1.0.0"
 
 
 class MayosrsDataset(datasets.GeneratorBasedBuilder):
```

Comment:
The class is called MayosrsDataset is that correct?

Reply:
Ah no, good catch!